test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -174,28 +174,44 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, tf2_only, cp
 
 
 # Helper function to test data service 
-def run_data_service_test(ec2_connection, tensorflow_training):
+def run_data_service_test(ec2_connection, tensorflow_training, cmd):
     ec2_connection.run('python3 -m pip install --upgrade pip')
-    ec2_connection.run('pip3 install tensorflow==2.4.0rc3')
+    ec2_connection.run('pip3 install tensorflow==2.4.0')
```

Comment:
nit: Please remove the `.0` so that the latest version of `tensorflow==2.4.x` is always installed.